### ISSUE TITLE
Undesired changes when performing diff on a simple model change

### ISSUE THREAD

**Issue by gabrielsmelchor** [otykier/TabularEditor#870]
**What are you trying to achieve?**
We author our model in Power BI Desktop, and use tabular editor v16.0 to serialize model changes, which we then perform diffs on. Most of the time this works well, but we occasionally get undesired diffs, like the one below:

![image](https://user-images.githubusercontent.com/38728420/122089686-3e73e980-cdc4-11eb-83fa-9a3d3507eb77.png)

This seems to be auto generated. We are wondering what we are doing to create these undesired changes (we have 700+). Is it a setting? Can it be avoided?

**Comment by otykier** [comment]
These annotations are not created by Tabular Editor. Are you working on a composite (DirectQuery over AS) model by any chance?

**Comment by gabrielsmelchor** [comment]
@otykier Your responsiveness is nothing short of legendary. We are working on an import model, with partitions in Power Query. Our prior change was hiding columns (including the one in the screenshot). Perhaps this was generated by Power BI when we hit the column?

**Comment by otykier** [comment]
> @otykier Your responsiveness is nothing short of legendary. We are working on an import model, with partitions in Power Query. Our prior change was hiding columns (including the one in the screenshot). Perhaps this was generated by Power BI when we hit the column?

Could be. As you may know, modifying column properties through external tools is not supported, so I guess this is one example of the consequences of performing unsupported modeling operations. I’ll ask around to see if I can learn more on what these annotations do.

**Comment by gabrielsmelchor** [comment]
@otykier Yes, we were aware of that limitation. We did modify the column properties directly in Power BI. I have tried my best to Google any clues. If you find anything that would be greatly appreciated. Thank you!
